mne/preprocessing/stim.py

```diff
@@ -67,3 +72,139 @@ def eliminate_stim_artifact(raw, events, event_id, tmin=-0.005,
         elif mode == 'window':
             raw[picks, first_samp:last_samp] = data * window[np.newaxis, :]
     return raw
+
+
+def fix_stim_artifact_raw(raw, events, event_id, tmin=-0.005,
```

Comment:
I thought the signature would be very different but indeed it is juste the events that we need extra for raw. If you confirm yes I think we could have a single function.